<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSettings</class>
 <widget class="QDialog" name="SystemSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>334</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>System settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="chkTimeCut">
         <property name="text">
          <string>Time cut:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnTimeLimit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string>min</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>720</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="chkCapacityCut">
         <property name="text">
          <string>Capacity cut:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnCapacityLimit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string>mAh</string>
         </property>
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="chkTempCut">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Temp. cut-off:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spnTempLimit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string>Â°C</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSystemBuzzer">
           <property name="text">
            <string>System buzzer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkKeyBuzzer">
           <property name="text">
            <string>Key buzzer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lblMessage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Loading settings</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>SystemSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
